app/models/timdex.rb

```diff
@@ -4,14 +4,110 @@
 class Timdex
   HTTP = GraphQL::Client::HTTP.new(ENV.fetch('TIMDEX_GRAPHQL', '')) do
     def headers(*)
-      { 'User-Agent': 'MIT Libraries Client' }
+      {
+        'User-Agent': 'MIT Libraries Client',
+        Origin: ENV.fetch('TIMDEX_UI_ORIGIN', 'http://localhost:3000')
```

Comment:
I don't think we should need to set Origin since we are processing the data in Rails and not in the UI directly. I could be wrong, but if you haven't seen an actual need for this in the UI I think removing it removes some potential confusion as to what is going on.